tools/clang-tidy-plugin/test/u8-path.cpp

```diff
@@ -145,9 +58,6 @@ void test_cases()
     // CHECK-MESSAGES: [[@LINE-1]]:54: warning: Construct `fs::path` by passing UTF-8 string to `fs::u8path` to ensure the correct path encoding.
     fs::path path_op_join_str_literal_path = ".." / path_u8_pchar;
     // CHECK-MESSAGES: [[@LINE-1]]:46: warning: Construct `fs::path` by passing UTF-8 string to `fs::u8path` to ensure the correct path encoding.
-    std::filesystem::operator/( "foo", "bar" );
-    // CHECK-MESSAGES: [[@LINE-1]]:33: warning: Construct `fs::path` by passing UTF-8 string to `fs::u8path` to ensure the correct path encoding.
-    // CHECK-MESSAGES: [[@LINE-2]]:40: warning: Construct `fs::path` by passing UTF-8 string to `fs::u8path` to ensure the correct path encoding.
```

Comment:
Is it removed because `std::filesystem` is not defined? If so it should be changed to `fs::` so `operator/` is still tested.

Reply:
I tried that but there was no suitable definition of `operator/` in `fs::`.  I think we can't know reliably where it's defined, so I'm not sure we can expect to be able to call it in this way.

Reply:
It's defined here:
 https://github.com/CleverRaven/Cataclysm-DDA/blob/3d1627dab6da00a647dccd215552ce37cf16603e/src/third-party/ghc/filesystem.hpp#L628